ml/models/embed_dnn.py

```diff
@@ -115,7 +115,7 @@ def forward(
 
             # concatenate the float tensor and embedded tensor
             X_float = torch.cat((X_float, avg_embeddings), dim=1)
-            id_feature_col_idx += 1
+            id_feature_col_idx += 2
```

Comment:
there's the bug lol

Reply:
damn son, good catch. a literal off by one!